src/app/page.tsx

```diff
@@ -1,41 +1,41 @@
 import { Inter } from "@next/font/google";
-import styles from "./page.module.css";
+import { PageSpin } from "./page-spin";
 
 const inter = Inter({ subsets: ["latin"] });
 
-export default async function Home() {
-  if (process.env.HEARTBEAT_URL == null) {
-    throw new Error("HEARTBEAT_URL is not set");
-  }
+export default async function Home(props: {
+    searchParams?: { [key: string]: string | string[] | undefined };
+}) {
+    const forceOffline = props.searchParams?.offline === "true";
```

Comment:
Not gonna hold the PR because of it, but I was thinking maybe it would be nice to instead have `forceStatus` or something similar so both states could be changed no matter what the state currently is.